src/search/search.stories.ts

```diff
@@ -0,0 +1,25 @@
+import { storiesOf, moduleMetadata } from "@storybook/angular";
+import { withKnobs } from "@storybook/addon-knobs/angular";
+
+import { SearchModule } from "../";
+
+storiesOf("Search", module).addDecorator(
+	moduleMetadata({
+		imports: [SearchModule]
+	})
+)
+	.addDecorator(withKnobs)
+	.add("Basic", () => ({
+		template: `
+		<div style="width: 250px;">
+			<ibm-search [size]="'lg'"></ibm-search>
```

Comment:
`size` can just be set directly

```suggestion
			<ibm-search size="lg"></ibm-search>
```